### PR TITLE
Microverse 1st capstone,Thopic: The vitron community

### PR DESCRIPTION
**Pull request by camilevahviraki**
### **In this milestone we: **

- [ ] Added the `Index.html` file which contains all the sections pages.
- [ ] Added the `index.css` file to style the capstone.
- [ ] Added the `index.js` file to improve some dynamism to the page.
- [ ] Added images folder to the repo.
- [ ] Added cocogoose custom font-family to the repo.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,264 @@
...
+        </ul>
+        <!-- <img src="images/PngItem_1074369.png" alt="" class="world_img" /> -->
+      </nav>
+     
+    </header>
+
+    <section class="home_page" id="home_page">
+      <div class="section_intro">
+        <h2 class="Intro">"Hello, make it bounce with Vitron"</h2>
+        <h1 class="big_title">THE VITRON COMMUNITY SAYS HELLO.</h1>
```
Comment (gemmen29) [inline index.html:65]: - [ ] Kindly decrease the font size of this heading in the mobile version `home` page to match the Behance design and to get rid of the general scrolling problem.

| Your design | Behance design |
| --- | --- |
| image | image |

File: index.html [hunk 2]
```diff
@@ -0,0 +1,264 @@
...
+        <p class="p_description" id="p_desckA">
+          Vitron Is a new brand enterprise based on software and hardware
+          devellopement, It's a company wich is extended over 10 contries in the
+          world.We gonna celebrate our lancemnet of the new product in the
+          market
+          <span>The Vitron 2.1 subwoofer</span> in may 2022
+        </p>
+        <h2 class="date_conf">2022.7.20(Mon)~22(Tue)</h2>
+        <h3 class="adresse">@The Liberty Warehouse,Red Hook,Brooklyn.</h3>
```
Comment (gemmen29) [inline index.html:74]: - [ ] Kindly adjust this part to be more to the left to match the Behance design.

| Your design | Behance design |
| --- | --- |
| image | image |

File: index.html [hunk 3]
```diff
@@ -0,0 +1,264 @@
...
+          </div>
+          <div class="card">
+            <div class="image_wrapper">
+              <img src="images/City-Ferris-Wheel-icon.png" alt="" class="icon" />
+            </div>
+            <h3 class="title_card">V Ignite</h3>
+            <p class="p_card">
+              Introducing our products in market with a big impacts to concurencies.
+            </p>
+          </div>
```
Comment (gemmen29) [inline index.html:124]: Kindly increase the spacing between `h3` and `p` to match the Behance design.

| Your design | Behance design |
| --- | --- |
| image | image |

File: index.html [hunk 4]
```diff
@@ -0,0 +1,264 @@
...
+        <img src="images/Hamburger.png" alt="" class="hamburger" id="hamburger1"/>
+        <img src="images/x-icon.png" alt="" class="hamburger" id="hamburger2"/>
+        <ul class="ul_nav" id="ul_nav">
+          <li id="home"><a class="links_Nav" >Home</a></li>
+          <li id="about"><a class="links_Nav" >About</a></li>
+          <li><a class="links_Nav" >Join</a></li>
+          <li><a class="links_Nav" href="#">Program</a></li>
+          <li><a class="links_Nav" href="#">Sponsor</a></li>
+          <li><a class="links_Nav" href="#">News</a></li>
+        </ul>
```
Comment (gemmen29) [inline index.html:41]: - [ ] Kindly make the mobile menu take the full height of the page and stop scrolling when the menu opens for a better UI experience.

File: README.md [hunk 5]
```diff
@@ -1 +1,54 @@
...
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
```
Comment (gemmen29) [inline README.md:52]: - [ ] Kindly add an acknowledgment to [Cindy Shin](https://www.behance.net/adagio07) the author of the original design as required by the Creative Commons License in your README file.

File: index.html (hunk 1 above)
Comment (gemmen29) [inline index.html:65]: - [ ] Kindly decrease the font size of this heading in the mobile version `about` page and make it in the center to match the Behance design and to get rid of the general scrolling problem.

| Your design | Behance design |
| --- | --- |
| image | image |

File: index.html [hunk 6]
```diff
@@ -0,0 +1,264 @@
...
+        <div class="past_card" id="past_card1">
+          <div class="overlay_red"></div>
+          <h2 class="year_last">2010</h2>
+          <p class="p_past_card">The votronXPZ</p>
+        </div>
+        <div class="past_card" id="past_card2">
+          <div class="overlay_red"></div>
+          <h2 class="year_last">2015</h2>
+          <p class="p_past_card">The votronXPZplus</p>
+        </div>
```
Comment (gemmen29) [inline index.html:228]: - [ ] Kindly make these paragraphs displayed properly so all words appear and decrease the spacing between year and paragraph to match the Behance design.

| Your design | Behance design |
| --- | --- |
| image | image |

**Review by gemmen29 — CHANGES_REQUESTED** [review]
Hi @camilevahviraki, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight 💯 
- No linters errors ✔️ 
- Descriptive pull request ✔️ 
- Descriptive commit messages ✔️ 

## Required Changes ♻️
_Check the comments under the review._



## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by micheaol — APPROVED** [review]
# STATUS: APPROVED :green_circle: :clap: 

Hi @camilevahviraki , 

Great work on making the changes 👏 
In my opinion, your project is ready for the presentation!  There is nothing else to say other than... it's time to merge it :shipit: 

well-1
Good luck! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by camilevahviraki** [comment]
Thanks

**Comment by camilevahviraki** [comment]
### Hi there:wave:
I have updated the app and made it Clean and Dry in [this pull request](https://github.com/camilevahviraki/Capstone_first/pull/2), and [the live demo](https://capstone-first-vitron-events.netlify.app/)

**Pull request merged**